Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -249,10 +249,8 @@ static float toNSURLSessionTaskPriority(WebCore::ResourceLoadPriority priority)
         [mutableRequest _setPrivacyProxyFailClosedForUnreachableHosts:YES];
 #endif
 
-#if HAVE(NETWORK_CONNECTION_PRIVACY_STANCE)
     if ([mutableRequest respondsToSelector:@selector(_setPrivacyProxyFailClosed:)] && advancedPrivacyProtections.contains(WebCore::AdvancedPrivacyProtections::FailClosedForAllHosts))
```

Comment:
One thing that caught my eye while quickly scrolling through this PR is that `respondsToSelector` seems unneeded too.